packages/light-apps/src/AddAccount/Restore.tsx

```diff
@@ -2,7 +2,7 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import React from 'react';
+import React, { useState } from 'react';
 import { RouteComponentProps } from 'react-router-dom';
 import { Menu } from '@substrate/ui-components';
```

Comment:
I think this was originally my doing but while we're refactoring, @ should come before r